web/themes/custom/usagov/usagov.theme

```diff
@@ -50,4 +50,16 @@ function usagov_theme_suggestions_alter(array &$suggestions, array $variables, $
       $suggestions[] = 'views_view_list__sub_children';
     }
   }
+  if ($hook == 'field') {
+    $object = $variables['element']['#object'];
+    if (get_class($object) == 'Drupal\node\Entity\Node') {
+      if (in_array('field__body', $suggestions)) {
```

Comment:
Suggestion: $suggestions will also include "field__node__body" here. How about replacing all this with: 
`if (in_array('field__node__body', $suggestions)) {`

I could be all wet, but I think that will catch the same cases and eliminate the need to explicitly check that the current object is a Node.

Reply:
> For the hook to be called the body in the cms cannot be empty. Any information in the body will be overwritten by the twig file but the hook is only ever called if there is data to begin with in the body field in the cms.

This is interesting! At minimum, we should take advantage of that by replacing the content in the body with a note like "DO NOT EDIT: content supplied by twig file."